<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SymbolManager</class>
 <widget class="QDialog" name="SymbolManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Symbol Manager</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="tabSymFiles" >
      <attribute name="title" >
       <string>Symbol files</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Loaded symbol files:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QTreeWidget" name="treeFiles" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>File</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Last refresh</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnAddFile" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveFile" >
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAddrLabels" >
      <attribute name="title" >
       <string>Address labels</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Address labels:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupSlots" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>0</hsizetype>
             <vsizetype>0</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>Active slots</string>
           </property>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="chk00" >
                <property name="text" >
                 <string>0-0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk01" >
                <property name="text" >
                 <string>0-1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk02" >
                <property name="text" >
                 <string>0-2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk03" >
                <property name="text" >
                 <string>0-3</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="chk10" >
                <property name="text" >
                 <string>1-0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk11" >
                <property name="text" >
                 <string>1-1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk12" >
                <property name="text" >
                 <string>1-2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk13" >
                <property name="text" >
                 <string>1-3</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="chk20" >
                <property name="text" >
                 <string>2-0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk21" >
                <property name="text" >
                 <string>2-1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk22" >
                <property name="text" >
                 <string>2-2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk23" >
                <property name="text" >
                 <string>2-3</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="chk30" >
                <property name="text" >
                 <string>3-0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk31" >
                <property name="text" >
                 <string>3-1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk32" >
                <property name="text" >
                 <string>3-2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk33" >
                <property name="text" >
                 <string>3-3</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupSegments" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>5</hsizetype>
             <vsizetype>5</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title" >
            <string>Active segments</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLineEdit" name="txtSegments" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>5</hsizetype>
                <vsizetype>1</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnAddSymbol" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemoveSymbol" >
           <property name="text" >
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <widget class="QTreeWidget" name="treeLabels" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Symbol</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Type</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Value</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Slots</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Segments</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Code range</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Source</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnReloadFiles" >
       <property name="text" >
        <string>Reload all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>SymbolManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>571</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
